libkbfs/keybase_daemon.go

```diff
@@ -13,10 +13,10 @@ import (
 	"github.com/keybase/client/go/logger"
 )
 
-// KeybaseServiceFn defines a constructor for a KeybaseService
-type KeybaseServiceFn func(config Config, params InitParams, ctx Context, log logger.Logger) (KeybaseService, error)
+// keybaseDaemonCn is the default
```

Comment:
Changed to `keybaseDaemon`. I think I was going for "Constructor functions" like how golang puts `Fn` at the end of types that are functions. Mostly cause I didn't want to use "Factory" or "Builder".